# Завдання 1

# First Notebook

In [1]:
print("Hello! Jupyter")

Hello! Jupyter


# Завдання 2

# Результат розрахунків суми та різниці

In [10]:
a = 15
b = 7
sum_ab = a + b
diff = a - b
print("сума =", sum_ab)
print("різниця =", diff)

сума = 22
різниця = 8


In [11]:
a = 15
b = 7
prod = a * b
frac = a / b
print("добуток =", prod)
print("частка =", frac)

добуток = 105
частка = 2.142857142857143


# Завдання 3

# Звіт із продажів
### У цьому розділі ми рахуємо підсумки за простими данними

In [1]:
sales = [120, 135, 100, 150, 90, 200, 170]
total = sum(sales)
average = total / len(sales)
maximum = max(sales)
minimum = min(sales)

print("Загальна кількість продажів:", total)
print("Середнє значення:", average)
print("Максимальне значення:", maximum)
print("Мінімальне значення:", minimum)

Загальна кількість продажів: 965
Середнє значення: 137.85714285714286
Максимальне значення: 200
Мінімальне значення: 90


| Показник | Значення |
|----------|----------|
| Загальна кількість продажів | 965 |
| Середнє значення | 137.86 |
| Максимальне значення | 200 |
| Мінімальне значення | 90 |

<table style="width: 80%; text-align: center;">
    <tr style="background-color: #888; color: white;">
        <th style="border: 1px solid black; padding: 8px;">Показник</th>
        <th style="border: 1px solid black; padding: 8px;">Значення</th>
    </tr>
    <tr>
        <td style="border: 1px solid black; padding: 8px;">Загальна кількість продажів</td>
        <td style="border: 1px solid black; padding: 8px;">965</td>
    </tr>
    <tr>
        <td style="border: 1px solid black; padding: 8px;">Середнє значення</td>
        <td style="border: 1px solid black; padding: 8px;">137.86</td>
    </tr>
    <tr>
        <td style="border: 1px solid black; padding: 8px;">Максимальне значення</td>
        <td style="border: 1px solid black; padding: 8px;">200</td>
    </tr>
    <tr>
        <td style="border: 1px solid black; padding: 8px;">Мінімальне значення</td>
        <td style="border: 1px solid black; padding: 8px;">90</td>
    </tr>
</table>